src/os/ui/datetime/datecontrol.js

```diff
@@ -101,13 +105,23 @@ os.ui.datetime.DateControlCtrl = function($scope) {
    */
   this['disabled'] = false;
 
+  /**
+   * If the duration is relative to the current date.
+   * @type {boolean}
+   */
+  this['relativeDuration'] = false;
+
   // take over updating the timeline controller
   this.assumeControl();
 
   // watch for changes to start/end dates
   $scope.$watch('dateControl.startDate', this.onStartDateChanged_.bind(this));
   $scope.$watch('dateControl.endDate', this.onEndDateChanged_.bind(this));
 
+  $scope.$on('startDate.userSelected', function(e) {
```

Comment:
Could you create a more formal handler for this? Something like `this.onStartDateSelected_`? The JSDoc type for Angular events is `angular.Scope.Event`, for the function's docs.

Reply:
Ah right, we even talked about that this morning! Thanks for being vigilant.